### DISCUSSION TITLE
Date Trigger and social media actions

### DISCUSSION THREAD

**Discussion by gzqn** [Budibase/budibase#4350] [Ideas]
It would be useful to have a trigger that operated on a date,  or a formula for a date, and preferably on a time as well.

It would also be useful to have an action (or many) that posted to social media.

**Comment by mjashanks** [comment]
We have a CRON trigger, which you can setup to run on an interval. Have you tried it?

You could then setup a REST API connection to your social media.

**Comment by jmxt3** [comment]
I'm also trying to accomplish that but with no luck. I need to execute a Cron Job every hour but only 8 Times. How I can restrict the frequency of the CRON trigger? Is it possible to suspend the automation after 8 times?

**Reply by mjashanks** [reply]
Hey @jmxt3 - you could potentially achieve this by creating a table that counts the executions. So, your trigger would look something like
1. cron trigger
2. query rows from table - probably need to order by the most recent, and limit to 1
3. filter - continue if rows.0.count (count on the first row) < 9
4. whatever steps need to happen
5. update the row - set count = count + 1
